style.css

```diff
@@ -383,3 +393,124 @@ header {
   background: #6070ff;
   color: #fff;
 }
+
+.nav-desktop {
+  display: none;
+}
+
+@media all and (min-width: 768px) {
+  /* ===============================*\
+           Header
+*\=============================== */
+
+  .nav-desktop {
+    display: block;
+  }
+
+  .menu-desktop {
+    list-style: none;
+    display: flex;
+    gap: 20px;
+  }
+
+  header {
+    padding: 0;
+  }
+
+  header .navbar .menu-desktop a {
+    font-size: 15px;
+    line-height: 20px;
+    font-style: normal;
+    font-weight: 500;
+    color: #344563;
+  }
+
+  header .navbar .menu-desktop a:hover {
+    color: #6070ff;
+  }
+
+  .hamb-bu {
+    display: none;
+  }
+
+  .navbar {
+    padding: 15px 142px;
+  }
+
+  /* ===============================*\
+           headline
+*\=============================== */
+
+  #headline {
+    background-image: url("images/Header-Desktop.svg");
+    justify-content: center;
+    padding: 0 110px;
+  }
+
+  #headline h1 {
+    font-size: 48px;
+  }
+
+  #headline p {
+    font-size: 20px;
+  }
+
+  /* ===============================*\
+           Work
+*\=============================== */
+  .portfolio-snapshot {
+    flex-direction: row;
+  }
+
+  #work {
+    padding: 142px;
+    margin: 0;
+  }
+
+  .img {
+    width: 100%;
+    cursor: pointer;
+  }
+
+  .order {
+    order: 2;
+  }
+
+  /* ===============================*\
+           About me
+*\=============================== */
+
+  #about-section {
+    padding: 114px 142px;
+    display: flex;
+    gap: 30px;
+  }
+
+  .about {
+    width: 50%;
+    padding: 0;
+  }
+
+  .language {
+    flex-direction: row;
+    width: 50%;
+  }
+
+  .classification-2 {
+    flex-direction: row;
+  }
+
+  .classification-2 li {
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  /* ===============================*\
+           Contact me
+*\=============================== */
+
+  .wrapper {
+    max-width: 800px;
+    margin: 0 auto;
+  }
+}
```

Comment:
Please ensure that your project sticks as much as possible to Figma Template provided. Kindly make sure that the submit button in the desktop version is centered. :+1: :smiley: :smiley: Also please make sure that the width of the text and the inputs are the same as provided in the Figma template.

Your design
![image](https://user-images.githubusercontent.com/98363075/200099324-b426bc77-9931-4d45-96a4-b5cf10d9e4aa.png)


Figma template
![image](https://user-images.githubusercontent.com/98363075/200099223-d2cde662-9e1c-4117-801b-be34353355e1.png)